### PR TITLE
*: make the scene of stale read clearer

### PR DESCRIPTION
**Pull request by nolouch**
Signed-off-by: nolouch <nolouch@gmail.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: stale-read.md [hunk 1]
```diff
@@ -5,21 +5,17 @@ summary: 介绍 Stale Read 功能和使用场景。
 
 # Stale Read 功能的使用场景
 
-本文档介绍 Stale Read 的使用场景。Stale Read 是一种读取历史数据版本的机制，读取 TiDB 中存储的历史数据版本。使用 Stale Read 功能，你能从指定时间点或时间范围内读取对应的历史数据，从而避免数据同步带来延迟。
-
-在内部实现上，TiDB 通过 Stale Read 可以从任意一个副本上读取到该指定时间点或时间范围内尽可能新的数据，并在这个过程中始终保证数据的一致性约束。
+本文档介绍 Stale Read 的使用场景。Stale Read 是一种读取历史数据版本的机制，读取 TiDB 中存储的历史数据版本。通过 Stale Read 功能，你能从指定时间点或时间范围内读取对应的历史数据，从而避免数据同步带来延迟。当使用 Stale Read 时，TiDB 默认会随机选择一个副本来读取，因此能利用上所有副本。如果你的应用程序不能容忍读到非实时的数据，请不要使用 stale read，不然读到的数据可能不是最新写入成功的数据。
```
Comment (TomShawn) [inline stale-read.md:8]: 这个写进下面的警告里会不会更醒目？

Comment (nolouch) [reply stale-read.md:8]: 下面主要是说明实验特性吧，不要说两件事情？

Comment (TomShawn) [reply stale-read.md:8]: 好吧

Comment (TomShawn) [reply stale-read.md:8]:
```suggestion
本文档介绍 Stale Read 的使用场景。Stale Read 是一种读取历史数据版本的机制，读取 TiDB 中存储的历史数据版本。通过 Stale Read 功能，你能从指定时间点或时间范围内读取对应的历史数据，从而避免数据同步带来延迟。当使用 Stale Read 时，TiDB 默认会随机选择一个副本来读取数据，因此能利用所有副本。如果你的应用程序不能容忍读到非实时的数据，请勿使用 Stale Read，否则读到的数据可能不是最新成功写入的数据。
```

**Review by TomShawn — APPROVED** [review]

**Review by NingLin-P — APPROVED** [review]

**Pull request merged**